code/go/0chain.net/blobbercore/handler/object_operation_grpc_handler.go

```diff
@@ -96,3 +101,72 @@ func (b *blobberGRPCService) DownloadFile(ctx context.Context, req *blobbergrpc.
 
 	return convert.DownloadFileResponseCreator(resp), nil
 }
+
+func (b *blobberGRPCService) WriteFile(ctx context.Context, req *blobbergrpc.UploadFileRequest) (*blobbergrpc.UploadFileResponse, error) {
+
```

Comment:
Blobber handler should call http handler. We should just write conversion functions to convert grpc inputs to http inputs and http outputs to grpc outputs.